src/scrabble.js

```diff
@@ -1,5 +1,165 @@
-function scrabble() {
-  // write code here
+const scoreLibrary = [
+  { letters: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'], score: 1 },
+  { letters: ['D', 'G'], score: 2 },
+  { letters: ['B', 'C', 'M', 'P'], score: 3 },
+  { letters: ['F', 'H', 'V', 'W', 'Y'], score: 4 },
+  { letters: ['K'], score: 5 },
+  { letters: ['J', 'X'], score: 8 },
+  { letters: ['Q', 'Z'], score: 10 }
```

Comment:
Nice way of assigning the scores to the letters. Tell me more 👀
I also used an object but like this:
"const letterScoring = {
A: 1
E: 1..."
Yours looks cleaner

Reply:
Thanks :) It is really useful to do structure like me it is just array where I put objects inside each object there is key letters also array where I puted letters and another key it is score where I puted score for each letter from previous key "letters"
Is is good because when you make loop for search letter and score for this letter this structure will work faster and you will write less code for search function.

Reply:
Thank you for explaining that to me - this is really helpful